packages/shared-ux/page/solution_nav/src/with_solution_nav.styles.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { css } from '@emotion/react';
+import { euiCanAnimate, EuiThemeComputed } from '@elastic/eui';
+
+export const WithSolutionNavStyles = (euiTheme: EuiThemeComputed<{}>) => {
```

Comment:
nit: I don't think this should start with a capital... it's not a React component, or a class.

Reply:
It is the convention we had used so far, eg:
https://github.com/elastic/kibana/blob/main/packages/shared-ux/button_toolbar/src/buttons/icon_button_group/icon_button_group.styles.ts#L11

https://github.com/elastic/kibana/blob/main/packages/shared-ux/card/no_data/impl/src/no_data_card.styles.ts#L11

Reply:
Happy to change it, but would prefer if we stayed consistent

Reply:
I will merge this now, but lmk if you think it needs to be changed.